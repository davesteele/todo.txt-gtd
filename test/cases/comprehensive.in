# BProj
#
# This is BProj

1st task @computer +BProj
second task @pending

Wandering task @computer +AProj

# AProj
Figure out what AProj is @computer

# _None

Make a CProj @computer +CProj
Do an AProj @computer +AProj
Lonely task @computer
Do a CProj @computer +CProj
