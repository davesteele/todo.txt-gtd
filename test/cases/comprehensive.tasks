
@computer
1st task @computer +BProj
Wandering task @computer +AProj
Figure out what AProj is @computer
Make a CProj @computer +CProj
Do an AProj @computer +AProj
Lonely task @computer
Do a CProj @computer +CProj

@pending
second task @pending
