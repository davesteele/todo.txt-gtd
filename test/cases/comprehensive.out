# AProj
Figure out what AProj is @computer +AProj

Wandering task @computer +AProj
Do an AProj @computer +AProj

# BProj
#
# This is BProj

1st task @computer +BProj
second task @pending +BProj


# CProj
#

Make a CProj @computer +CProj
Do a CProj @computer +CProj

# _None

Lonely task @computer
